src/main/java/pl/patrycja/ox/gameflow/AutomaticTests.java

```diff
@@ -0,0 +1,82 @@
+package pl.patrycja.ox.gameflow;
+
+import pl.patrycja.ox.GameSettings;
+import pl.patrycja.ox.Sign;
+import pl.patrycja.ox.ui.UI;
+
+import java.util.List;
+
+class AutomaticTests extends Mode {
+
+    AutomaticTests(UI ui) {
+        super(ui);
+    }
+
+    @Override
+    public List<Player> createPlayers() {
+        Player firstPlayer = new Player("Henio", Sign.CROSS);
+        Player secondPlayer = new Player("Olaf", Sign.NAUGHT);
+        return List.of(firstPlayer, secondPlayer);
+    }
+
+    @Override
+    public void settings(String[] inputArrayParameters) {
+        gameSettings = setup(inputArrayParameters);
+        Sequence sequence = new Sequence(gameSettings, inputArrayParameters[2]);
+        sequence.generateSequence();
+    }
+
+    private GameSettings setup(String[] inputArrayParameters) {
+        checkCorrectInputData(inputArrayParameters);
+        //TODO: generate draw sequence!!
+        return updateValuesInSettings(inputArrayParameters);
+    }
+
+    private GameSettings updateValuesInSettings(String[] inputArrayParameters) {
+        GameSettings.GameSettingsBuilder gameSettings = GameSettings.builder();
+
+        int boardSize = Integer.parseInt(inputArrayParameters[0]);
+        int unbrokenLine = Integer.parseInt(inputArrayParameters[1]);
+        int maxBoardSize = Math.max(boardSize, unbrokenLine);
+        int minUnbrokenLine = Math.min(boardSize, unbrokenLine);
+        gameSettings.boardSize(maxBoardSize);
+        gameSettings.unbrokenLine(minUnbrokenLine);
+
+        int towards = Integer.parseInt(inputArrayParameters[2]);
```

Comment:
Please move this line under `int maxBoardSize = Math.max(boardSize, unbrokenLine);` that all `inputArrayParameters` values are in one place.

Reply:
Ok, I will do it.